src/Approximations/iterative_refinement.jl

```diff
@@ -131,29 +135,26 @@ function refine(Ω::PolygonalOverapproximation, i::Int)::Tuple{LocalApproximatio
 end
 
 """
-    tohrep(Ω::PolygonalOverapproximation{N})::AbstractHPolygon where {N<:Real}
+    tohrep(Ω::PolygonalOverapproximation{N})::AbstractHPolygon{N} where N<:Real
 
 Convert a polygonal overapproximation into a concrete polygon.
 
 ### Input
 
-- `Ω`   -- polygonal overapproximation of a convex set
+- `Ω` -- polygonal overapproximation of a convex set
 
 ### Output
 
 A polygon in constraint representation.
 """
-function tohrep(Ω::PolygonalOverapproximation{N})::AbstractHPolygon where {N<:Real}
-    p = HPolygon{N}()
-    for ai in Ω.approx_list
-        addconstraint!(p, LinearConstraint(ai.d1, dot(ai.d1, ai.p1)))
-    end
-    return p
+function tohrep(Ω::PolygonalOverapproximation{N}
```

Comment:
good! could a `Algorithm` comment that there is no need for using `addconstraint!` because the constraints `Ω.constraints` are known to be already sorted? 

(if the `HPolygon` constructor has at some later point a default sorting of the constraints, we'd like to turn that flag off here)